ext/opentelemetry-ext-cloud-trace/src/opentelemetry/ext/cloud_trace/__init__.py

```diff
@@ -0,0 +1,236 @@
+# Copyright OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Cloud Trace Span Exporter for OpenTelemetry."""
+
+import logging
+from typing import Sequence, Dict, Any, List
+
+import opentelemetry.trace as trace_api
+from google.cloud.trace_v2 import TraceServiceClient
+from google.cloud.trace_v2.proto.trace_pb2 import AttributeValue
+from opentelemetry.sdk.trace import Event
+from opentelemetry.sdk.trace.export import Span, SpanExporter, SpanExportResult
+from opentelemetry.util import types
+
+logger = logging.getLogger(__name__)
+
+# Max length is 128 bytes for a truncatable string.
+MAX_LENGTH = 128
+
+
+class CloudTraceSpanExporter(SpanExporter):
+    """Cloud Trace span exporter for OpenTelemetry.
+
+    Args:
+        client: Cloud Trace client.
+        project_id: project_id to create the Trace client.
```

Comment:
I think we should reword this a bit. Perhaps something like `project_id: ID of the Cloud project that will receive the trace data.` (I used [this doc](https://cloud.google.com/trace/docs/reference/v1/rest/v1/projects.traces/list#path-parameters) as inspiration.)

Also, we should probably switch the order of the args in the docstring to match the args to the function.